ui/content.html

```diff
@@ -38,7 +38,7 @@
                                 ${renderThumbnail(content, displayOptions)}
                                 <div>
                                     <div>
-                                        <h1 class="oae-threedots">${content.displayName|encodeForHTML}</h1>
+                                        <span class="oae-clip-text oae-threedots">${content.displayName|encodeForHTML}</span>
```

Comment:
I consider the primary clip title to be the main title for the page (user name, group name, content item name, etc.) and think that it should remain an `h1`.